### PR TITLE
 GameShow functionality and test coverage completed

### PR DESCRIPTION
**Pull request by henrike32**
GameShow functionality
TestCoverage
Added GameShow.css on style folder

### CODE REVIEW

File: src/styles/GameShow.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  min-height: 100vh;
+  padding: 1rem;
+  background-image: url('https://images.unsplash.com/photo-1505506874110-6a7a69069a08?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=987&q=80');
+  background-size: cover;
+  background-position: center;
+}
+
+.show-game-card-container {
+  /* padding-top: 5%; */
+  /* margin-top: 5%; */
```
Comment (SunkissedQueen) [inline src/styles/GameShow.css:14]: if this code will not be used be remove

File: src/pages/GameShow.js [hunk 2]
```diff
@@ -1,13 +1,57 @@
...
+              <Button class=" text-white  bg-info" >Edit</Button>
+            </CardBody>
+          </Card>
+        </div>
+      )}
+    </main>
+  );
+};
+
+export default GameShow;
```
Comment (SunkissedQueen) [inline src/pages/GameShow.js]: The trello stated that you would have a button to go back to the game index.

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Please see comments

File: src/pages/GameShow.js (hunk 2 above)
Comment (henrike32) [reply src/pages/GameShow.js]: The button is in the dropdown menu , "View All" games.

**Pull request merged**